test/train-sets/ref/help.stdout

```diff
@@ -424,8 +434,9 @@ Contextual Bandit with Action Dependent Features:
                         (no clipping).
   --cb_type arg         contextual bandit method to use in {ips, dm, dr, mtr, 
                         sm}. Default: mtr
-Contextual Bandit Options:
-  --cb arg              Use contextual bandit learning with <k> costs
+Deprecated: Contextual Bandit Options:
```

Comment:
Do we also want to mark the setup function for this reduction with `VW_DEPRECATED` to discourage it's usage?

Reply:
VW_DEPRECATED might be for API public facing functions, but not entirely sure. Maybe @jackgerrits can chime in. The current strategy was going to be:
- figure out how to message this change via --help
- add legacy on cb on ` Enabled reductions: gd, scorer, cb (legacy), ...`

Since those two items are the way end users interact with VW. A possible third item would be to hide completely from --help the legacy cb but might be too radical. Same hide() from #2710